tests/unit/test_report_junit.py

```diff
@@ -80,10 +80,10 @@ def _compare_xml_node(tree_path: List[str], expected: xml.dom.Node, actual: xml.
 
     for (expected_name, expected_value), (actual_name, actual_value) in zip(
             expected_attributes, actual_attributes):
-        assert expected_name == actual_name, (f"Attribute mismatch at {tree_path_joined}: "
-                                              f"expected {expected_name}=\"{expected_value}\"")
-        assert expected_value == actual_value, (f"Attribute mismatch at {tree_path_joined}: "
-                                                f"found {actual_name}=\"{actual_value}\"")
+        assert expected_name == actual_name, (f'Attribute mismatch at {tree_path_joined}: '
+                                              f'expected {expected_name}="{expected_value}"')
```

Comment:
It would be cool to standardize single or double quotes by default.
Q000 rule(currently disabled) can be configured to preferred single/double quote style.